src/masoniteorm/commands/ShellCommand.py

```diff
@@ -0,0 +1,198 @@
+import subprocess
+import os
+import re
+import shlex
+from collections import OrderedDict
+
+from ..config import load_config
+
+from .Command import Command
+
+
+class ShellCommand(Command):
+    """
+    Connect to your database interactive terminal.
+
+    shell
+        {--c|connection=default : The connection you want to use to connect to interactive terminal}
+        {--s|show=? : Display the command which will be called to connect}
+    """
+
+    shell_programs = {
+        "mysql": "mysql",
+        "postgres": "psql",
+        "sqlite": "sqlite3",
+        "mssql": "sqlcmd",
+    }
+
+    def handle(self):
+        resolver = load_config(self.option("config")).DB
+        connection = self.option("connection")
+        if connection == "default":
+            connection = resolver.get_connection_details()["default"]
+        config = resolver.get_connection_information(connection)
+        if not config.get("full_details"):
+            self.line(
+                f"<error>Connection configuration for '{connection}' not found !</error>"
+            )
+            exit(-1)
+
+        command, env = self.get_command(config)
+
+        if self.option("show"):
+            cleaned_command = self.hide_sensitive_options(config, command)
+            self.comment(cleaned_command)
+            self.line("")
+
+        # let shlex split command in a list as it's safer
+        command_args = shlex.split(command)
+        try:
+            subprocess.run(command_args, check=True, env=env)
+        except FileNotFoundError:
+            self.line(
+                f"<error>Cannot find {config.get('full_details').get('driver')} program ! Please ensure you can call this program in your shell first.</error>"
```

Comment:
I was getting this error for every connection but I guess because I didn't setup a mysql shell program

Reply:
oh okay, but it's weird, I think that when I install mysql or postgres server, those shell programs were bundled. I did not installed them separately. I installed those with Homebrew.

Reply:
Anyway in the meantime you can test with sqlite.
SQLite has a shell !